src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -591,6 +590,142 @@ def get_default_realm(self) -> str:
         raise DashboardException(msg='Default realm not found.',
                                  http_status_code=404,
                                  component='rgw')
+        
+    def get_multiste_info(self):
+        defaults_info = {}
+        multisite_info = []
+        all_realms_info = []
+        zonegroup_list = []
+        all_zonegroup_info = []
+        default_realm_id = ''
+        default_zonegroup_id = ''
+        rgw_realm_list_cmd = ['realm', 'list'] 
+        try:
+            _, out, err = mgr.send_rgwadmin_command(rgw_realm_list_cmd)
```

Comment:
no error handling is a bit scary.

Reply:
A way to deal with errors could be 
```suggestion
            exit_code out, err = mgr.send_rgwadmin_command(rgw_realm_list_cmd)
            if exit_code > 0:
              raise DashboardError...
            # do something with out
```

Reply:
Done